### ISSUE TITLE
Starting 'Server Timings' then 'Query Plan' then disabling 'Query Plan' locks out user queries

### ISSUE THREAD

**Issue by greggyb** [DaxStudio/DaxStudio#857]
**Describe the bug**
Exactly what the title says. After enabling 'Server Timings', enabling 'Query Plan', and disabling 'Query Plan', you cannot submit a query via F5 or clicking the 'Run' button in the 'Home' ribbon.

Note: tested only with a local PBIX.

**To Reproduce**
1. Open a new query tab
2. Click 'Home' > 'Server Timings'
3. Click 'Home' > 'Query Plan'
4. Click 'Home' > 'Query Plan'
5. Try to run a query

**Expected behavior**
The query should run

**Additional context**
This happens both if you enable 'Query Plan' before the trace for 'Server Timings' is started or if you let the trace start for 'Server Timings'.

Output is below. Comments are preceded with '#' and give context to the line below them.

1. Interleaved start
```
2022-05-23 00:08:46		Establishing Connection
# click 'Server Timings'
2022-05-23 00:08:50		Waiting for Trace to start
# click 'Query Plan'
2022-05-23 00:08:50		Reconfiguring Trace
2022-05-23 00:08:51		Query Trace Started
# click 'Query Plan'
2022-05-23 00:08:54		Reconfiguring Trace
2022-05-23 00:08:56		Query Trace Started
# 'Run' and F5 both fail to produce results. Cache can be cleared.
```

2. Serial start
```
2022-05-23 00:11:22		Establishing Connection
# click 'Server Timings'
2022-05-23 00:11:27		Waiting for Trace to start
2022-05-23 00:11:27		Query Trace Started
# click 'Query Plan'
2022-05-23 00:11:29		Reconfiguring Trace
2022-05-23 00:11:31		Query Trace Started
# click 'Query Plan'
2022-05-23 00:11:33		Reconfiguring Trace
2022-05-23 00:11:35		Query Trace Started
# no queries for you!
```

**Comment by greggyb** [comment]
Note: interleaving other actions does not seem to make any difference. I.e., 'Server Timings', run query, 'Query Plan', run query, 'Query Plan' fails in the same way.

Note2: Disabling 'Server Timings' after we get to hung queries makes queries run again.

Note3: Same error mode is seen if you do 'Query Plan', 'Server Timings', 'Server Timings'. It appears that starting both in any order and then disabling one gets you into a no-query state. Further stopping the second leads to queries working again.

**Comment by greggyb** [comment]
Originally tested on 2.16.2. Updated to 2.17.3 and still presenting apparently identically.

**Comment by dgosbell** [comment]
I think this should be fixed in the current release 3.0.6. As part of the v3 update I altered the tracing infrastructure so that each trace button has a dedicated server side trace. This is a tiny bit more resource intensive on the server, but means starting and stopping one trace type does not interfere with other traces.

@greggyb - Can you confirm this is working for you now?

**Comment by dgosbell** [comment]
closing due to lack of response
